src/graphics/thing.js

```diff
@@ -512,7 +512,7 @@ class Thing {
      * Describes the element for use with screen readers.
      */
     describe() {
-        return `A ${this.type} at ${this.x}, ${this.y}. Colored: ${this.color}.`;
+        return `A ${this.type} at ${this.x}, ${this.y}. Colored: ${this.color.toUpperCase()}.`;
```

Comment:
I'm not sure if this is the best fix, but if you use the voiceover for your example which has the color `#560bad`, it says the word "bad" when reading it. By adding the uppercase it reads each letter. I did notice that it does read `#F72585` as "F seventy two thousand...," so I'm not sure how to fix that. The downside is that if you put in a color like `white` it comes out as `WHITE`. With the mac voiceover it still reads the word, but I think I read somewhere that different voiceovers might interpret that as "W...H...I...T...E". Think this would take more testing to figure out and we might end up wanting to write some sort of describe function for color.

Reply:
interesting, i think that a describe function for color makes a lot of sense. ideally we'd be able to parse hex values and give them some sort of meaning ("red" instead of "#FF0000"), but that seems difficult.

alternatively we could move towards enforcing strings as colors instead of hex strings. all valid CSS colors are valid for the context, so you can say `circle.color = 'red'` or `'aqua'` or whatever and it works. 

as a strategy for naming colors, we could compute the distance to a named color, so `#b30000` would have a distance of `b8 - b3, 00 - 00, 00 - 00` from "dark red" `#b80000`

Reply:
Here's a quick draft of that https://codehs.com/sandbox/andy/color-namer

Reply:
This is super cool. I wonder what other people do and what's actually the most useful. I think for beginners just using names makes a lot of sense, but at some point we may need/want to handle the ability to get the hex, rgba, and hsl values for a color.

Reply:
I think our HSL implementation and RGB -> hex in this library are broken, but we can definitely support those. I've been meaning to revisit Color soon.